src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,84 @@
+package duke.task;
+
+import java.time.LocalDate;
+
+/**
+ * A class that represent deadline task
+ */
+public class Deadline extends Task {
+    private String name;
+    private boolean isDone;
+    private String time;
+
+    /**
+     * constructor
+     * @param name
+     * @param isDone
+     * @param time
+     */
+    public Deadline(String name, boolean isDone, String time) {
+        super();
+        this.name = name;
+        this.isDone = isDone;
+        this.time = time;
+    }
+
+    @Override
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public boolean isDone() {
+        return isDone;
+    }
+
+
+    /**
+     * get time
+     * @return
```

Comment:
Some javadocs seem to be missing, consider filling them in when you have time?

Reply:
thanks for reminding